### ISSUE TITLE
When Dub hides the background, D+ can bring it back

### ISSUE THREAD

**Issue by diremuffin** [DubPlus/DubPlus#8]
When you go into mobile mode (by shrinking the window enough), the background disappears:
![image](https://cloud.githubusercontent.com/assets/25775750/23232507/503d949c-f900-11e6-94c2-5a20e64f744f.png)
I assume this is standard functionality. 
The new thing is that if in D+ you hide the background:
![image](https://cloud.githubusercontent.com/assets/25775750/23232525/6392b8b0-f900-11e6-9dd1-5072dc4d3766.png)
Then unhide it:
![image](https://cloud.githubusercontent.com/assets/25775750/23232533/6c580f72-f900-11e6-9598-5737a0d59e94.png)

the background comes back.

I don't know if this is intended behavior, but I wanted to draw your attention to it. Basically, it might be worth considering changing it so that the slider automatically shows "Hide Background" when the window is shrunk so that Dub hides the background?
If that's not possible, however, this is obviously not a big problem. However, I'd consider it unexpected behavior and I wanted to make sure you knew about it. :D

**Comment by FranciscoG** [comment]
notes for myself:

I have to override that media screen which keeps the bg on at 48em and over
```css
@media screen and (min-width: 48em)
  .backstretch {
    display: block;
  }
}
.backstretch {
    display: none;
}
```

**Comment by FranciscoG** [comment]
deleted my previous comment because I misread your issue. I know why this is happening and can fix.  thanks for reporting!

**Comment by FranciscoG** [comment]
fixed in #9
